include/nighthawk/client/options.h

```diff
@@ -29,7 +29,7 @@ class Options {
   virtual uint32_t connections() const PURE;
   virtual std::chrono::seconds duration() const PURE;
   virtual std::chrono::seconds timeout() const PURE;
-  virtual std::string uri() const PURE;
+  virtual absl::optional<std::string> uri() const PURE;
```

Comment:
This might be a good place to document why it is optional.

Reply:
Added comment